src/scripts/run_floodscan_pipeline.py

```diff
@@ -0,0 +1,59 @@
+import argparse
+from datetime import datetime
+
+import pandas as pd
+from src.config.settings import load_pipeline_config
+from src.pipelines.floodscan_pipeline import FloodScanPipeline
+
+
+def parse_arguments(base_parser):
+    today = datetime.today()
+    yesterday = today - pd.DateOffset(days=1)
+    parser = argparse.ArgumentParser(parents=[base_parser])
+    parser.add_argument(
+        "--start-date",
+        "-s",
+        help="""Start date to retrieve and process archival FloodScan data. 
+        Format: '%Y%m%d""",
```

Comment:
Similar to the other comment, could we keep the date formatting consistent with `%Y-%m-%d`?